video_worker/api_communicate.py

```diff
@@ -52,9 +54,7 @@ def run_veda(self):
 
         self.veda_token = generate_apitoken.veda_tokengen()
         if self.veda_token is None:
-            ErrorObject().print_error(
-                message='VEDA API Conn Fail:\nInvalid Setup/Method'
-            )
```

Comment:
We're using them for logs, but, actually raising them as errors is maybe a better choice. If you want to just log the error for now, that's totally fine.